dm/worker/worker_test.go

```diff
@@ -184,6 +192,7 @@ func (t *testServer2) TestTaskAutoResume(c *C) {
 	var subtaskCfg config.SubTaskConfig
 	c.Assert(subtaskCfg.DecodeFile("./subtask.toml", true), IsNil)
 	c.Assert(err, IsNil)
+	subtaskCfg.Mode = "full"
```

Comment:
why this is needed?

Reply:
because i don;t want task into sync unit 😂, i don't mock all db connection for now